### DISCUSSION TITLE
token expired or revoked

### DISCUSSION THREAD

**Discussion by scidmail** [GAM-team/got-your-back#449] [Q&A]
I have been using GYB for quite some time with no issues. I was using a older version, but since it was working, I didn't want to make any changes.  I'd run a batch file daily with this command: gyb --email myemailaddress@gmail.com --search "newer_than:7d" --action backup

Mid January I started receiving this error.  ERROR: ('invalid_grant: Token has been expired or revoked.', {'error': 'invalid_grant', 'error_description': 'Token has been expired or revoked.'})

I assumed that I needed to update GYB for something google had changed. I downloaded the latest windows version 1.8 and began the reinstall process. Once completed I tried again, but received the same error. I looked in the discussions for the error, but the only one I found was old and didn't have an answer. 

I'd really appreciate if anyone can help me. I'm not exactly well versed in how to work in the google console, so please be kind, use small words, and explain step by step if you can help.

Thank you in advance.

**Comment by asavageiv** [comment]
I was able to fix this by removing the `.cfg` file in my `config` folder, then trying the backup again. It repeated the OAuth flow giving access to my account and then worked from there.
